images/Dockerfile

```diff
@@ -54,6 +54,7 @@ RUN adduser --disabled-password --gecos "" --uid 1001 runner \
 WORKDIR /home/runner
 
 COPY --chown=runner:docker --from=build /actions-runner .
+COPY --from=build /usr/local/lib/docker/cli-plugins/docker-buildx /usr/local/lib/docker/cli-plugins/docker-buildx
```

Comment:
do you also need `--chown=runner:docker`?

Reply:
I don't think that's necessary. It worked fine in my local testing.

Also, in our custom images, we've been using a `root:root` owned `docker-buildx` file without issues.